src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { ErrorTypes } from '../constants';
+import { Loader } from '../components/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorType, setErrorType] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const errorNotification = errorType === ErrorTypes.UPLOAD
```

Comment:
just move it to the return

Reply:
Make it this way but in this case we get nested conditional rendering